### DISCUSSION TITLE
Tradeoffs between different fw?

### DISCUSSION THREAD

**Discussion by AJolly** [pvvx/ATC_MiThermometer#493] [Q&A] [answered]
Whats the tradeoffs between the different firmware versions available? I'm using HA if that matters. I see:

Support for Zigbee firmware is in the [ZigbeeTLc](https://github.com/pvvx/ZigbeeTLc) repository.

Support for BLE-Zigbee firmware is in the [BZdevice](https://github.com/pvvx/BZdevice) repository.

There is an alternative firmware from [Ivan Belokobylskij](https://github.com/devbis) for [LYWSD03MMC to work in Zigbee 3.0 mode](https://github.com/devbis/z03mmc).

**Comment by pvvx** [comment, marked as answer]
There are no significant ones, except for [battery consumption](https://github.com/pvvx/ZigbeeTLc/issues/37#issuecomment-1937844181).
Simple BLE is the most economical.
